### ISSUE TITLE
Squelch devise deprecation warning

### ISSUE THREAD

**Issue by colinxfleming** [DCAFEngineering/dcaf_case_management#1666]
Thanks for creating an issue! Please fill out this form so we can be sure to have all the information we need, and to minimize back and forth.

* **What are we trying to do?**

CI is throwing a lot of warnings to this effect:

```
DEPRECATION WARNING: [Devise] `DeviseHelper.devise_error_messages!`
is deprecated and it will be removed in the next major version.
To customize the errors styles please run `rails g devise:views` and modify the
`devise/shared/error_messages` partial.
 (called from block in _app_views_devise_sessions_new_html_erb___1413956147758747842_47447809537020 at /home/circleci/abortioneering/app/views/devise/sessions/new.html.erb:10)
```

* **What feature or behavior is this required for?**

being able to keep devise up to date, which is really important since it's our auth library!

* **How could we solve this issue? (Not knowing is okay!)**

I think we have two things to do here:

* bump devise to the latest version
* regenerate the views via `rails g devise:views`
* re-add any custom stuff we put into those views, making sure we don't re-add the deprecated junk.

* **Anything else?**

nah

**Comment by rudietuesdays** [comment]
I can grab this tonight!

**Comment by colinxfleming** [comment]
@rudietuesdays still working on this one?

**Comment by colinxfleming** [comment]
freeing this up!

**Comment by mdworken** [comment]
Closed via #1737
